src/global_input.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from typing import Optional
+
+from arcade import key
+from pyglet.input import Controller
+from pyglet.window.key import KeyStateHandler
+
+from player_input import VirtualButton
+
+
+class GlobalInput:
+    def __init__(self, keys: KeyStateHandler, controller: Optional[Controller]) -> None:
```

Comment:
I like the implementation of global inputs. This will be great later for things like menus and settings